lib/phoenix_html/tag.ex

```diff
@@ -224,9 +225,21 @@ defmodule Phoenix.HTML.Tag do
   defp csrf_token_tag(opts, extra) do
     case Keyword.pop(opts, :csrf_token, true) do
       {true, opts} ->
-        {opts, extra <> ~s'<input name="_csrf_token" type="hidden" value="#{get_csrf_token()}">'}
+        {opts, extra <> ~s'<input name="#{@csrf[:param]}" type="hidden" value="#{get_csrf_token()}">'}
       {false, opts} ->
         {opts, extra}
     end
   end
+
+  @doc """
+  Generates a meta tag with CSRF information.
+
+  ## Tag attributes
+    `content`      - a valid csrf token
+    `csrf-param`   - a request parameter where expected csrf token
+    `method-param` - a request parameter where expected a custom HTTP method
+  """
+  def csrf_meta_tag do
+    raw ~s'<meta charset="UTF-8" name="csrf-token" content="#{get_csrf_token()}" csrf-param="#{@csrf[:param]}" method-param="#{@csrf[:method]}">'
```

Comment:
Please use the proper `tag` to generate this to ensure we won't be vulnerable to xss attacks in the future in case somebody decides add new fields.